routers/api/v1/repo/migrate.go

```diff
@@ -113,12 +113,16 @@ func Migrate(ctx *context.APIContext, form auth.MigrateRepoForm) {
 		gitServiceType = api.GithubService
 	}
 
+	if form.Mirror && setting.Repository.DisableMirrors {
+		ctx.Error(http.StatusForbidden, "MirrorsGlobalDisabled", fmt.Errorf("mirrors are global disabled"))
```

Comment:
To keep it consistent with UI
```suggestion
		ctx.Error(http.StatusForbidden, "MirrorsGlobalDisabled", fmt.Errorf("the site administrator has disabled mirrors"))
```